README.md

```diff
@@ -26,15 +26,6 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
-Contributing
-------------
-
-Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
-When you send your request, make `master` the destination branch on the
-[DTK repository](https://github.com/ORNL-CEES/DataTransferKit).
-
-Your PR must pass DTK's tests. We enforce these guidelines with Jenkins CI.
-
```

Comment:
I think it's a good idea to have that CONTRIBUTING.md file on the repo but I would still reference it from our README.md.  At least a
```
Contributing
------------
Please see [CONTRIBUTING.md](CONTRIBUTING.md).
```
like in the example from [factory_girl_rails](https://github.com/thoughtbot/factory_girl_rails) or
even better something like in [puppet](https://github.com/puppetlabs/puppet).